components/date-picker/createPicker.tsx

```diff
@@ -68,6 +68,12 @@ export default function createPicker(TheCalendar: React.ComponentClass): any {
       };
     }
 
+    componentDidUpdate(prevProps: PickerProps) {
+      if ('open' in prevProps && prevProps.open) {
```

Comment:
`if (prevProps.open)`. Just doing this is enough

Reply:
it had updated